lib/feed.xml

```diff
@@ -1,26 +1,57 @@
 <?xml version="1.0" encoding="utf-8"?>
-{% capture url_base %}{% if site.url %}{{ site.url | append: site.baseurl }}{% else %}{{ site.github.url }}{% endif %}{% endcapture %}
-<rss version="2.0"
-	xmlns:dc="http://purl.org/dc/elements/1.1/"
-	xmlns:atom="http://www.w3.org/2005/Atom"
-	>
-<channel>
+{% if site.url %}
+  {% assign url_base = site.url | append: site.baseurl %}
+{% else %}
+  {% assign url_base = site.github.url %}
+{% endif %}
+<feed xmlns="http://www.w3.org/2005/Atom">
+  <generator uri="http://jekyllrb.com" version="{{ jekyll.version}}">Jekyll</generator>
+  <link href="{{ url_base }}{{ page.url }}" rel="self" type="application/atom+xml" />
```

Comment:
Added a space, as all the other tag closures had a space, but not strongly opinionated.